build_all.bat

```diff
@@ -0,0 +1,76 @@
+rem "build from source on windows"
+
+@echo off
+echo requires `gcc` in your PATH

```

Comment:
I'm not familiar with batch, but does this get echoed no matter what?

Wouldn't it be better to echo it only if ``gcc`` isn't in the PATH?